pandas/core/base.py

```diff
@@ -990,11 +990,17 @@ def factorize(self, sort=False, na_sentinel=-1):
         from pandas.core.algorithms import factorize
         return factorize(self, sort=sort, na_sentinel=na_sentinel)
 
-    def searchsorted(self, key, side='left'):
-        """ np.ndarray searchsorted compat """
+    def searchsorted(self, key, side='left', sorter=None):
```

Comment:
just realized that ``Series.searchsorted`` has a doc-string (and includes the ``sorter`` arg, so:

can you move the ``Series.searchsorted`` doc-string to ``pandas/core/base.py`` (and update it with the warning that you wrote below).

Then since everything else inherits this, I think ou can drop the docs strings (and they will be inherited).